apps/docs/docs/components/pagination/_default.component.ts

```
@@ -0,0 +1,63 @@
+  // ng-doc-ignore-line
+  selector: 'flowbite-demo-navbar-default',
+  imports: [PaginationComponent],
+  // ng-doc-ignore-line
+  templateUrl: './_default.component.html',
+})
+export class FlowbiteDefaultComponent {
+  private readonly _testDatas: IdNameDto[] = [];
+
+  readonly currentPage = signal(1);
+  readonly pageSize = signal(10);
+  readonly testDatas = signal(PagedResult.empty<IdNameDto>());
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)